### DISCUSSION TITLE
寻人委托

### DISCUSSION THREAD

**Discussion by QingZaoChuan** [share121/inter-knot#4917] [委托]
我找不到我的爱人了，就在前阵子，我和他起了争执，他非要去参加什么演武仪典，我觉得太危险了，想拦着他，但他直接走了，哎，他就是这个性子，总之，请求大家接下这个委托，帮我找到他，对了以下是他的照片，哎，干什么不好，非要去打架![StarRail_Image_1724214929](https://github.com/user-attachments/assets/95d54235-7104-47e2-89c5-e67ccc6007e3)

**Comment by Liruijia666** [comment]
这个委托，我接。。。。。。我没有波提欧，接不了一点

**Comment by ZM-zm921** [comment]
好像在演武仪典上看到过这位…，当时他打败了一个红发小子

**Comment by xingyuwuyuan** [comment]
我记得他，演武仪典差点闹出人命

**Comment by tingyu919** [comment]
当时他和彦卿小哥的决赛正打到关键时刻，不知道为什么突然就停手认输了，听说好像不想被公司的人缠住
